pkg/output/output.go

```diff
@@ -30,13 +30,23 @@ func (o *Output) AddException(message error) {
 	o.exceptions = append(o.exceptions, message)
 }
 
-// SetFailures sets failures as a bulk update
-func (o *Output) SetFailures(failures []string) {
+// SetEgressFailures sets egress endpoint failures as a bulk update
+func (o *Output) SetEgressFailures(failures []string) {
 	for _, f := range failures {
 		o.failures = append(o.failures, handledErrors.NewEgressURLError(f))
 	}
 }
 
+// GetEgressFailures gets egress endpoint failures
+func (o *Output) GetEgressFailures() (failures []error) {
```

Comment:
same as above: consider using `cli.output.Parse()`

Reply:
I used it for asserting the unit test. same as above..